src/amo/components/SearchForm.scss

```diff
@@ -0,0 +1,36 @@
+@import "~core/css/inc/vars";
+
+.SearchForm-form {
+  background: rgba(255, 255, 255, 0.2);
+  border: 0;
+  display: flex;
+
+  .SearchForm-query {
+    background: url('~amo/img/icons/magnifying-glass.svg') 18px 50% no-repeat;
+    border: 0;
+    color: $header-font-color;
+    font-size: $font-size-m;
+    flex-grow: 1;
+    height: 3rem;
+    margin: 0 auto;
+    max-width: 320px;
```

Comment:
Where did this number come from? Chrome adds a big outline on focus and it looks weird that the input isn't 100% width. It also bothers me a little that it isn't the full width, I feel like it should be.

![screenshot 2016-09-27 11 36 30](https://cloud.githubusercontent.com/assets/211578/18882882/a92f59b0-84a6-11e6-8b41-848ccf544967.png)